Api/Database/db-provider.js

```diff
@@ -0,0 +1,17 @@
+var config = require('./knexfile');
+
+var dbProvider = function dbProvider() {
```

Comment:
Tutaj też bym zrobiłe funtion DbProvider(), metody to prototype i module.exports = new DbProvider()